insights/parsers/oracfg.py

```diff
@@ -71,6 +72,9 @@ class OracleConfig(Parser):
             string
 
     """
+    def __init__(cls, *args, **kwargs):
```

Comment:
I expected to see `self` here instead of `cls`.

Reply:
Hi @PaulWay - How do you think about the comments that Bob commented above?